workitem/workitem_revision_repository_blackbox_test.go

```diff
@@ -39,12 +40,12 @@ func (s *workItemRevisionRepositoryBlackBoxTest) TestStoreRevisions() {
 		// modify the workitem
 		wi.Fields[workitem.SystemTitle] = "Updated Title"
 		wi.Fields[workitem.SystemState] = workitem.SystemStateOpen
-		wi, err := s.repository.Save(s.Ctx, wi.SpaceID, *wi, fxt.Identities[1].ID)
+		wi, err := s.repository.Save(s.Ctx, wi.SpaceID, *wi, fxt.Identities[1].ID, uuid.NewV4())
```

Comment:
@surajssd Tests are missing! You should add a test to assert that the `revisionID` that you pass to save this workitem is the same as the revision ID of the retrieved revision.